_events/5.markdown

```diff
@@ -1,18 +1,23 @@
 ---
 layout: event
 title:  "2019-11-19 Call"
-date:   2019-11-19 9:00
+date:   2019-11-19 10:00
 file_list:
-  # google drive links
-summary: This is a summary of the call.
+- https://docs.google.com/presentation/d/18bT4-wVe3PheeiMLKRvQxpXoCeSjzH3Hj-Axn6V2VFE/edit#slide=id.g78fcc90ba3_2_110
```

Comment:
We don't need the dash `-`, just the indent like in https://raw.githubusercontent.com/hms-dbmi/x-consortium/master/_example_templates/0101.markdown

Reply:
i don't think the dash is necessary still